website/views.py

```diff
@@ -66,10 +78,20 @@ def get_messages(request, *args, **kwargs):
 #  length (minimum - 5, max - 1000). In addition, there should be non-null user_id.
 #  To insert, use methods from the same helper, construct_message and insert_web_message.
 #  As a result return JSON with response 200 - ok (google how it should look like.)
-# No need to test if that works right now. We will check it later.
+
 
 def insert_message(request, *args, **kwargs):
-    print(request)
+    user_id = int(kwargs.get('user_id'))
```

Comment:
check for None first:
[http://127.0.0.1:8000/api/messages/](http://127.0.0.1:8000/api/messages/)

Should return error404 and not an exception.